bitnami/kong/templates/declarative-config-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.kong.declarativeConfig }}
```

Comment:
This restriction is too weak: the deployment will fail if both `declarativeConfigCM` and `declarativeConfig` are provided because `{{ include "kong.declarativeConfigMap" . }}` will return the name of an already existing configmap 

Either change the helper to avoid both values to be provided or use a more resilient condition so this resource is not created.

Reply:
Added a more strict restriction